presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.h

```diff
@@ -31,7 +31,7 @@ class PartitionAndSerializeNode : public velox::core::PlanNode {
       uint32_t numPartitions,
       velox::RowTypePtr outputType,
       velox::core::PlanNodePtr source,
-      velox::core::PartitionFunctionFactory partitionFunctionFactory)
```

Comment:
Thanks. It would be nice to extract velox update and related changes into a separate PR to avoid coupling these this functionality changes.

Reply:
Sure, I just need this to make a custom build with @tanjialiang for some internal experiments.